sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/Helpers/SonarAnalysisContextTest.cs

```diff
@@ -44,7 +44,6 @@ private class TestSetup
             new TestSetup { Path = @"TestCases\GetHashCodeEqualsOverride.cs", Analyzer = new GetHashCodeEqualsOverride() },
             new TestSetup { Path = @"TestCases\DisposeNotImplementingDispose.cs", Analyzer = new DisposeNotImplementingDispose() },
             new TestSetup { Path = @"TestCases\ClassShouldNotBeAbstract.cs", Analyzer = new ClassShouldNotBeAbstract() },
-            new TestSetup { Path = @"TestCases\ClassName.cs", Analyzer = new ClassAndMethodName() }

```

Comment:
Not sure what this list of tests is for. I can't keep the test in the list because it requires the partial class to be also present.